emscripten.py

```diff
@@ -1228,9 +1229,9 @@ def create_exports(exported_implemented_functions, in_table, function_table_data
     asm_runtime_funcs.append('setAsync')
   all_exported = exported_implemented_functions + asm_runtime_funcs + function_tables(function_table_data, settings)
   if settings['EMULATED_FUNCTION_POINTERS']:
-    all_exported = list(set(all_exported).union(in_table))
+    all_exported = set(all_exported).union(in_table)
```

Comment:
`all_exported` was a list before, then this if might or might not turn it into a `set`, which doesn't seem good. why not leave it as a `list` (we sort it later in this patch anyhow)?

Reply:
Can't think of a reason, so done 👍